### PR TITLE
wire up the new integration test creds

### PR DESCRIPTION
**Pull request by jgrillo-grapl**
### Which issue does this PR correspond to?
 NA

### What changes does this PR make to Grapl? Why?

This PR updates the infrastructure configurations to account for the changes in https://github.com/grapl-security/confluent-cloud-infrastructure/pull/3.

The upshot of this is now we have Kafka credentials specifically for the integration test consumers, and we use them instead of trying to piggy-back on e.g. the pipeline-ingress credentials.

### How were these changes tested?
Automated tests

### CODE REVIEW

File: pulumi/grapl/__main__.py [hunk 1]
```diff
@@ -196,14 +196,19 @@ def main() -> None:
...
+    integration_tests_kafka_credentials = kafka.service_credentials("integration-tests")
+    pulumi.export(
+        "integration-tests-kafka-sasl-username",
+        integration_tests_kafka_credentials.apply(lambda c: c.api_key),
+    )
     pulumi.export(
-        "pipeline-ingress-kafka-consumer-group-name",
-        pipeline_ingress_kafka_consumer_group_name,
+        "integration-tests-kafka-sasl-password",
+        integration_tests_kafka_credentials.apply(lambda c: c.api_secret),
```
Comment (wimax-grapl) [inline pulumi/grapl/__main__.py:211]: Purely hypothetically, couldn't these be declared in the `integration_tests/__main__.py`? That way you could skip the whole export-import song and dance, plus it's better namespacing.

Comment (wimax-grapl) [reply pulumi/grapl/__main__.py:211]: Not saying you gotta do this, but just raising it as a possibility.

File: nomad/integration-tests-new.nomad [hunk 2]
```diff
@@ -36,19 +36,19 @@ variable "pipeline_ingress_healthcheck_polling_interval_ms" {
   description = "The amount of time to wait between each healthcheck execution."
 }
 
-variable "pipeline_ingress_kafka_consumer_group_name" {
+variable "integration_tests_kafka_consumer_group_name" {
```
Comment (wimax-grapl) [inline nomad/integration-tests-new.nomad:39]: I'd argue you don't need to necessarily add these integration_tests_ prefixes given the filename, but not that important. I'm fine with the env vars having it, though.

**Review by wimax-grapl — APPROVED** [review]

File: pulumi/grapl/__main__.py (hunk 1 above)
Comment (jgrillo-grapl) [reply pulumi/grapl/__main__.py:211]: Yeah, I think they can. It'll take constructing the appropriate `Kafka()` but it seems doable. I'll give that a shot.

Comment (jgrillo-grapl) [reply pulumi/grapl/__main__.py]: fixed

**Pull request merged**